spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/metrics/SparkBigQueryConnectorMetricsUtils.java

```diff
@@ -60,6 +61,62 @@ public static void postConnectorVersion(SparkContext sparkContext, String connec
     }
   }
 
+  public static void postWriteSessionMetrics(
```

Comment:
please document why Class.forName() is used instead of compilation dependency

Reply:
Done